src/IO/Archives/LibArchiveReader.cpp

```diff
@@ -185,6 +240,8 @@ class LibArchiveReader::Handle
     }
 
     const String path_to_archive;
+    std::unique_ptr<StreamInfo> read_stream = nullptr;
+    const IArchiveReader::ReadArchiveFunction archive_read_function;
```

Comment:
`const` members are generally best avoided, because they inhibit the automatic generation of copy / move assignment. In this case, this may not be significant.